test/bench/gen_bench.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+#
+# See ./util/gen_bench.sh --help for documentation.
+
+
+#The first line of the file "benchmark_version" holds the
+#version number, see docs in that file.
+BENCH_VERSION=$(head -n 1 benchmark_version)
+
+show_usage () {
+  cat <<EOF
+Usage: $0 [-h|--help] [<branch>|<commit>|<tag>]
+EOF
+}
+
+show_help () {
+  echo ""
+  show_usage
+  echo ""
+  cat <<EOF
+./util/build_core.sh
```

Comment:
I guess this is supposed to show help for gen_bench.sh and not util/build_core.sh